README.md

```diff
@@ -49,6 +49,48 @@ If you'd like to protect your Sentry install with SSL/TLS, there are
 fantastic SSL/TLS proxies like [HAProxy](http://www.haproxy.org/)
 and [Nginx](http://nginx.org/). You'll likely want to add this service to your `docker-compose.yml` file.
 
+OR do the following things to enable auto redirect to https behind a reverse proxy
+
+Uncomment the following in `docker-compose.yml` nginx directive
+```
+ #expose:
+ #  - '443'
+ ports:
+ #  - '443:443'
+   - '$SENTRY_ONPREMISE_PORT:80/tcp'
```

Comment:
For this, we need #512 merged first, right?

Reply:
correct this is currently branched off of #512 and i've targeted back into 512 at this time for merging